frontend/file/build.gradle.kts

```diff
@@ -41,9 +41,6 @@ sourceSets["main"].java { srcDir(additionalSourceDirs) }
 
 val generateMacheteParser =
   tasks.withType<GenerateParserTask> {
-    // See https://github.com/JetBrains/gradle-grammar-kit-plugin/issues/89
-    outputs.cacheIf { true }
```

Comment:
Shameless piggyback 😅 caching wasn't really working for this tasks, they were somehow executed much more often than expected 🤔